### PR TITLE
quoting numeric value

### PR DESCRIPTION
**Pull request by billimek**
Attempting to install chart 1.3.0 with vault.reconciliationTime set to some value (in my case, `300`) results in a helm deployment error:

```
helm chart v1.EnvVar.Value: ReadString: expects \\\" or n, but found 3
```

Based on [this issue](https://github.com/helm/helm/issues/4775) it seems that numeric ENV values need to be quoted, hence this PR.

### CODE REVIEW

**Comment by billimek** [comment]
I'm happy to update this PR to quote _all_ of the env variables, as suggested by https://helm.sh/docs/developing_charts/#quote-strings-don-t-quote-integers

Just let me know, @ricoberger

File: charts/vault-secrets-operator/Chart.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ maintainers:
   - name: Rico Berger
     url: https://ricoberger.de
 name: vault-secrets-operator
-version: 1.3.0
+version: 1.3.1
```
Comment (ricoberger) [inline charts/vault-secrets-operator/Chart.yaml]: Can you revert this change please. The version bump is handled via the Makefile.

Comment (billimek) [reply charts/vault-secrets-operator/Chart.yaml]: done!

**Comment by ricoberger** [comment]
Thanks for the fixing this. I think it does not hurt to do that for all environment variables.

**Comment by ricoberger** [comment]
Thanks again. I will merge and release it immediately.
